### PR TITLE
added template sag-mws-applatform and sag-designer-applatform

### PR DESCRIPTION
**Pull request by ravishankarsegu**
I have added below templates for review
template sag-mws-applatform which will install Application Platform on MWS
template sag-designer-applatform which will install Application Platform perspective on designer

### CODE REVIEW

**Comment by sergeipogrebnyak** [comment]
Please see my inline comments for the changed files.

You need to add the README.md for designer template
And please update the sag-is-applatform template as well

Thanks
Sergei

**Comment by ravishankarsegu** [comment]
I have added the README.md file for designer template and also updated the sag-is-appplatform template

**Comment by ravishankarsegu** [comment]
Change in the spelling of template folder. sag-is-appplatform to sag-is-applatform

File: templates/sag-mws-applatform/template.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+alias: sag-mws-appplatform
+description: Application platform on MWS
```
Comment (YanaSimeonova) [inline templates/sag-mws-applatform/template.yaml]: just test

**Pull request merged**